apps/openchallenges/edam-etl/src/main.py

```diff
@@ -1,16 +1,68 @@
-"""Extract, transform and load EDAM concepts"""
-
+import pandas as pd
+import requests
 from os import getenv
 
 # Get config from the environment variables
 oc_db_url = getenv("OC_DB_URL")
```

Comment:
nit: [PEP 8](https://peps.python.org/pep-0008/#constants) prefers constant variables to be in all caps, e.g.

```suggestion
OC_DB_URL = getenv("OC_DB_URL")
```

Reply:
updated constant variable naming